bubble_sort.py

```diff
@@ -0,0 +1,14 @@
+def bubbleSort(arr):
+    for i in range(len(arr)):
+        swapped = False
```

Comment:
I like this variable usage to ensure algorithm ends earlier. We should add some test cases later to ensure this does not cause issues though.

Reply:
Thanks for sharing your thoughts! I agree